Automattic/sensei
# Implement quiz REST fetching

### Changes proposed in this Pull Request

* Implements the REST API endpoint fetching according to the schema introduced in #3890.
* There were minor updates to the schema, mainly minor fixes.
* Question related code belong to a question-related class but I postponed doing it until we define the question endpoint. The method separation was done with that refactor in mind.

### Testing instructions

* Try the endpoint '/sensei-internal/v1/quiz/<quiz_id>' for various quizzes and observe that the returned data are correct.
* Observe that one teacher cannot access the quizzes created by another and that an admin can access everything.
